_visualisation/optional/metadata.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Metadata Overlays
+short_title: Metadata
+position: 2.05
+---
+
+Metadata Overlays is an [Optional Feature](/visualisation/optional/) of BOVS Diagrams. It allows further information about Parties to be shown as icons overlaid on their Nodes.
+{: .lead}
+
+
+## Metadata Types
+
+The types of metadata you wish to show are your choice; we envisage that these are likely to include properties such as:
+
+| Metadata Type              | Explanation
+| ----------------------------
+| Regime of Jurisdiction     | Which Regime (e.g. country) has jurisdiction over this Party?
```

Comment:
I think this can just be 'Jurisdiction' - and then "Which jurisdiction(s) is this Party registered or resident in?"

Reply:
Would require updating below also. 

Linked to changes to definition of Jurisdiction.